packages/ui-components/src/__tests__/VaultDetail.test.ts

```
@@ -162,7 +180,26 @@ describe('VaultDetail', () => {
-		const invalidateQueries = vi.spyOn(queryClient, 'invalidateQueries');
+		render(VaultDetail, {
+			props: defaultProps,
+			context: new Map([['$$_queryClient', queryClient]])
+		});
+
+		await waitFor(async () => {
+			const refreshButton = await screen.getByTestId('top-refresh');
+			await userEvent.click(refreshButton);
+			expect(invalidateTanstackQueries).toHaveBeenCalledWith(queryClient, [
+				'100',
+				QKEY_VAULT + '100'
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)